server/src/test/kotlin/org/javacs/kt/DebouncerTest.kt

```diff
@@ -12,7 +12,7 @@ class DebouncerTest {
 
     @Test fun callQuickly() {
         for (i in 1..10) {
-            debounce.submit {
+            debounce.submit { ->
```

Comment:
```suggestion
            debounce.submit {
```

I would leave out the arrow if the lambda has no arguments.

Reply:
There's an issue with this - as I added the new overload to submit, without the arrow it is not clear which overload should be invoked:
`e: .../KotlinLanguageServer/server/src/test/kotlin/org/javacs/kt/DebouncerTest.kt: (15, 22): Overload resolution ambiguity: 
public final fun submit(task: () -> Unit): Unit defined in org.javacs.kt.util.Debouncer
public final fun submit(task: (cancelCallback: () -> Boolean) -> Unit): Unit defined in org.javacs.kt.util.Debouncer
`

So I've resolved the ambiguity by adding an arrow (and then neglected to add a comment about it, sorry). I don't know if there's a trick that can be played, or if there's any point in keeping the original overload.

Reply:
I have removed the original overloads as they were not needed anymore and made some small stylistic changes in the implementation (see e999ec5bc4a15467ce0622bda84f97e5e2cff3df for details).